src/botPage/view/blockly/blocks/tools/webhook.js

```diff
@@ -0,0 +1,59 @@
+import { translate } from '../../../../../common/i18n';
+import { expectValue } from '../shared';
+
+Blockly.Blocks.webhook = {
+    init() {
+        this.jsonInit({
+            message0: translate('Webhook URL : %1'),
```

Comment:
You can remove the space after `Webhook URL`

```suggestion
            message0: translate('Webhook URL: %1'),
```